eth-bytecode-db/eth-bytecode-db-server/src/settings.rs

```diff
@@ -79,6 +85,19 @@ impl Default for SourcifySettings {
     }
 }
 
+#[derive(Debug, Default, Clone, Deserialize, PartialEq, Eq)]
+#[serde(default, deny_unknown_fields)]
+pub struct VerifierAllianceDatabaseSettings {
+    pub enabled: bool,
+    pub url: String,
```

Comment:
why not `url::Url` to add config validation ?)

Reply:
Actually, because that is how `sea-orm` handles connection url